src/index.d.ts

```diff
@@ -3816,6 +3838,28 @@ declare namespace admin.messaging {
     volume?: number;
   }
 
+  /**
+   * Represents Options for features provided by the FCM SDK for iOS..
+   */
+  interface ApnsFcmOptions {
+
+    /**
+     * The Label associated with the message's analytics data..
+     */
+    analyticsLabel?: string;
+  }
+
+  /**
+   * Represents Platform independent options for features provided by the FCM SDKs.
```

Comment:
Nit: I don't think this needs to be capitalized.

It's an open question whether this term should be hyphenated ("platform-independent") when used as a modifier, so I leave that to you :)